dw-hardened-ami/hardening.sh

```diff
@@ -170,7 +185,8 @@ echo "2.2.1.2 Ensure ntp is configured"
 echo "Exemption; Amazon Linux recommends chrony"
```

Comment:
Uh-oh...this comment isn't true anymore, and it looks like we've not hardened the NTP config just yet.